node/commons/compatibility-layer.ts

```diff
@@ -1,13 +1,25 @@
 import { distinct } from '../utils/object'
 import unescape from 'unescape'
+import { Checkout } from '../clients/checkout'
+import R from 'ramda'
```

Comment:
`ramda` is very heavy, so we always choose to try to use native functions or, if it is not possible, import only the necessary functions. can we do it here?

Reply:
I made a new commit so I only import the 4 functions I use from ramda.